tests/test_drs.py

```diff
@@ -81,6 +103,163 @@ def test_drs_get_no_default(client, user):
     settings["config"]["INDEX"]["driver"].config["ADD_PREFIX_ALIAS"] = True
 
 
+def test_timestamps(client, user):
```

Comment:
we are missing checks of DRS objects' `index_created_time` and `index_updated_time` in the tests

Reply:
Added these checks.